src/gtk/toga_gtk/widgets/button.py

```diff
@@ -17,8 +17,7 @@ def set_label(self, label):
         self.rehint()
 
     def set_enabled(self, value):
-        # self._impl.set_sensitive(value)
-        self.interface.factory.not_implemented('Button.set_enabled()')
+        super().set_enabled(value)
```

Comment:
Isn't this redundant? If you just delete this implementation, won't the default widget implementation be called?